mods/ra/maps/soviet-05/reinforcements_teams.lua

```diff
@@ -0,0 +1,80 @@
+sov1 = {"e2", "e2"}
+sov1Path = { StartPoint.Location, SovietBasePoint.Location }
+
+if Map.Difficulty == "Easy" then
+	ArmorReinfGreece = { "jeep", "1tnk", "1tnk" }
+else
+	ArmorReinfGreece = {"jeep", "jeep", "1tnk", "1tnk", "1tnk" }
+end
+InfantryReinfGreece = { "e1", "e1", "e1", "e1", "e1" }
+CrossroadsRainfPath = { ReinfNorthLandPoint.Location, RunnerRetreatPoint.Location, NearSovietBasePoint.Location }
+arty1reinf = { "e3", "e3", "e3", "arty", "arty" }
+
+AlliedInfantryTypes = { "e1", "e3" }
+AlliedTankTypes = { "jeep", "1tnk" }
+AlliedAttackPath = { GreeceBaseEntryPoint.Location, CrossroadPoint.Location, NWOrefieldPoint.Location, NearSovietBasePoint.Location }
+AlliedCrossroadsToRadarPath = { CrossroadPoint.Location, GreeceBaseEntryPoint.Location, GreeceBasePoint.Location }
+SouthReinfPath = { ReinfEastPoint.Location, SouthReinfPathPoint1.Location, SouthReinfPathPoint2.Location, SouthReinfPathPoint3.Location, SouthReinfPathPoint4.Location, SovietExpansionUnloadPoint.Location }
+NorthReinfPath = { ReinfEastPoint.Location, NorthReinfPathPoint1.Location, mcvGGUnloadPoint.Location }
+
+Ships = { }
+GreeceInfAttack = { }
+GGInfAttack = { }
+TankAttackGG = { }
+CoastGuardInf = { "e1", "e1", "e3", "e1", "e3" }
+SovExpansionPointGuard = {"2tnk", "2tnk", "e3", "e3", "e3"}
+
+ShipWaypoints = { EastofLittleIsland, WestofLittleIsland, SBNavalAttackPoint }
+InfantryWaypoints = { CrossroadPoint, NWOrefieldPoint, NearSovietBasePoint, SovietBasePoint }
+InfantryGGWaypoints = { PatrolPoint2, BetweenExpBasesPoint, NearSovietExpansionPoint }
+TanksGGWaypoints = { PatrolPoint2, BetweenExpBasesPoint, NearSovietExpansionPoint }
+
+Para = function()
+	local powerproxy = Actor.Create("powerproxy.paratroopers", false, { Owner = player })
+	local units = powerproxy.SendParatroopers(ParaPoint.CenterPosition, false, 32)
+	Utils.Do(units, IdleHunt)
```

Comment:
I'm not sure if it's a good idea to let the player's units run away to attack by themselves.